components/organism/nestedCheckboxes/src/index.js

```diff
@@ -0,0 +1,117 @@
+import React, {useState} from 'react'
+import PropTypes from 'prop-types'
+
+import MoleculeCheckboxField from '@s-ui/react-molecule-checkbox-field'
+
+const baseClass = 'sui-OrganismNestedCheckboxes'
+
+const noop = () => {}
+
+const OrganismNestedCheckboxes = ({
+  id,
+  labelParent = '',
+  onChangeParent = noop,
+  items = [],
+  join = false,
+  onChangeItem = noop,
+  fullCheckedStyledIcon: FullCheckedStyledIcon,
+  halfCheckedStyledIcon: HalfCheckedStyledIcon,
+  showItems: showItemsProp = true,
+  showItemsIcon: ShowItemsIcon,
+  hideItemsIcon: HideItemsIcon
+}) => {
+  const [showItems, setShowItems] = useState(showItemsProp)
+  const showToggleIcons = ShowItemsIcon !== null
+
+  const parentIsFullChecked = items.every(({checked}) => checked === true)
```

Comment:
IMHO `Boolean` variables starting with: is, are, have, should, etc. improve developer code readability.